### DISCUSSION TITLE
add documentation for running zksync node

### DISCUSSION THREAD

**Discussion by zxsimba** [zkSync-Community-Hub/zksync-developers#161] [Feature requests]
### Network

zkSync Era

### Provide a brief description of the feature you would like to see implemented.

Other chains (L1 and L2) have good documentation about running a node,
I found nothing useful for zksync

### Who does this feature request benefit the most (users, developers, protocol, etc.)?

node holders

**Comment by dutterbutter** [comment]
Hey @zxsimba 👋 

You can expect this to change in the future when we open the network up to decentralize network participants, for the meantime you can review the docs for running and developing zkSync Era node [here](https://github.com/matter-labs/zksync-era/tree/main/docs), and an external node [here](https://github.com/matter-labs/zksync-era/tree/main/docs/external-node).

**Comment by zxsimba** [comment]
Thanks, I didn't know that zksync is not decentralized yet.
But I am wondering if the source code or binaries of node is not released, how ankr provided an rpc interface to zksync?

**Reply by dutterbutter** [reply]
They are running an external node [here](https://github.com/matter-labs/zksync-era/tree/main/docs/external-node). You can run an external node that provides read-only privileges to the network today. You can also see the [source code](https://github.com/matter-labs/zksync-era/tree/main), **we are open-sourced entirely**.

**Comment by zxsimba** [comment]
Great, and the final questions:
`The network is not decentralized yet` : is it due to lack or incompleteness of consensus algorithm or its implementation?

Is there any estimate on when the network would be open to decentralization?

**Comment by inesarmetessa** [comment]
This feature request would benefit node holders on zkSync by providing comprehensive documentation on running a node, similar to what is available for other chains. This documentation would empower node operators to effectively manage and maintain their zkSync nodes, leading to a more robust and decentralized network infrastructure.

**Comment by dawn71v** [comment]
Enhanced documentation for running a zkSync node would benefit both developers and node operators by providing clear instructions and guidance on setting up and maintaining a node on the zkSync network.

**Comment by dunminecom** [comment]
It would be beneficial for node holders on the zkSync network to have comprehensive documentation on running a node.

**Comment by moyemv** [comment]
The user is requesting better documentation for running a node on zkSync, as they have been unable to find useful information. This feature request would benefit node holders. The user has not provided any additional details.
